src/ssl.c

```diff
@@ -10955,6 +11033,171 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
         return SSL_NOT_IMPLEMENTED;
     }
 
+    #if defined(WOLFCRYPT_HAVE_SRP) && !defined(NO_SHA256) \
+    && !defined(WC_NO_RNG)
+
+    int wolfSSL_CTX_set_srp_username(WOLFSSL_CTX* ctx, char* username)
+    {
+        int r = 0;
+        int srp_side = 0;
+        WC_RNG rng;
+        byte salt[SRP_SALT_SIZE];
+        static byte N[] = {
```

Comment:
Please use `const byte` not `static byte`. Same for g[] below. This will give the compiler more tips for optimizations. The static option would put this into a fixed location specific to this function. The const will allow it to read directly from ROM/Flash.

Reply:
Thank you for your suggestion.
I am not familiar with embedded system. so very helpful.
I change from static to const.